spec/requests/partner_requests_spec.rb

```diff
@@ -0,0 +1,96 @@
+require "rails_helper"
+
+RSpec.describe "Partner Requests Controller", type: :request do
+  context "when user authenticated" do
+    let!(:partner) { create(:partner, :verified) }
```

Comment:
I tend to [avoid the usage of `let` blocks](https://thoughtbot.com/blog/lets-not) in my rspec tests.

Reply:
Once I get home, I’m going to check this blog post, thx for the tip I really appreciate your sharing.